plumbing/transport/ssh/common.go

```diff
@@ -11,32 +11,22 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
-// New errors introduced by this package.
 var (
-	ErrAdvertistedReferencesAlreadyCalled = errors.New("cannot call AdvertisedReference twice")
-	ErrAlreadyConnected                   = errors.New("ssh session already created")
-	ErrAuthRequired                       = errors.New("cannot connect: auth required")
-	ErrNotConnected                       = errors.New("not connected")
-	ErrUploadPackAnswerFormat             = errors.New("git-upload-pack bad answer format")
-	ErrUnsupportedVCS                     = errors.New("only git is supported")
-	ErrUnsupportedRepo                    = errors.New("only github.com is supported")
+	errAlreadyConnected = errors.New("ssh session already created")
```

Comment:
It's an internal error that can should not happen calling only public methods. connect() is called in newFetchPackSession and nowhere else. The user cannot make call it twice.